include/psa/crypto_extra.h

```diff
@@ -2069,6 +2069,264 @@ static inline struct psa_pake_operation_s psa_pake_operation_init(void)
     return v;
 }
 
+typedef enum {
+/** An invalid key data format identifier. */
+    PSA_KEY_DATA_FORMAT_NONE,
+/**
+ *  DER or PEM encoded RSAPublicKey data structure as defined in RFC 8017
```

Comment:
The description of this identifier should mention that it is used to specify the key format in a call to `psa_import_key_ex()`.